README.md

```diff
@@ -1,56 +1,123 @@
+
+
+4. In the Midtrans - Accept Online Payment section, click Basic Settings and fill out the following fields:
+
+| Field                   | Description									                               |
+|-------------------------| ---------------------------------------------------------------------------|
+| Is Production           | Select whether you want to use a sandbox or production mode\.			|
+| Merchant ID             | Unique id of your Midtrans account for which the payments will be processed\.|
+| Sandbox \- ClientKey    | Used as an API key to be used for authorization sandbox environment on frontend API request/configuration\. So it is safe to put in your HTML / client code publicly\.    |
+| Sandbox \- ServerKey    | Used as an API key to be used for authorization sandbox environment while calling Midtrans API from the backend\. So keep it stored confidentially\.                      |
+| Production \- ClientKey | Used as an API key to be used for authorization production environment on frontend API request/configuration\. So it is safe to put in your HTML / client code publicly\. |
+| Production \- ServerKey | Used as an API key to be used for authorization production environments while calling Midtrans API from the backend\. So keep it stored confidentially                    |
+| Enable Snap redirect    | Change to Snap redirect mode, the default value is No\.			 |
+
+
+>Note: Access Keys are unique for every merchant. Server Key is secret, please always keep Server Key confidential.
+
+## Log options
+
+The plugins will store log file in directory `/var/log/midtrans`. The default value is on for request, notification and error log. Except Throw Exception, is off by default.
+![](https://user-images.githubusercontent.com/21098575/78235349-b9741b80-7502-11ea-9139-f5119193bc20.png "image_mag_config")
+
+
+## Config Plugins Payment Integration
+
+In the Midtrans Magento plugins we have 4 option to use Snap model payment method, with the following description:
+
+
+1. **Snap payment integration**
+    
+    This is the default Snap for Midtrans Magento plugins, Snap payment will auto-enabled when install the Midtrans plugins. Midtrans will show the available payment method on the Snap payment screen.
+
+2. **Specific Payment integration | Optional** 
+    
+    Specific payment it’s design as optional for specific payment that specify in Allow payment method field, Midtrans Snap will show directly payment method on the Snap screen.
+
+3. **Online Installment payment integration | Optional**
+
+    Online Installment payment it’s design as optional for Installment where the Card Issuer and Acquiring Bank is the same entity (e.g: BNI Card and BNI Acquiring bank).
+
+4. **Offline Installment payment integration | Optional**
+
+    Offline payment it’s desing as optional for Installment where the Card Issuer and Acquiring Bank don't have to be same entity (e.g: BNI Card and Mandiri Acquiring Bank)
```

Comment:
Change according to suggested fix on commented-review of  `midtrans_snap_installment_payment.xml` file

Reply:
done